R/PythonPkgWrapperUtils.R

```diff
@@ -25,15 +25,55 @@ defineEnum <- function(assignEnumCallback, name, keys, values) {
   assignEnumCallback(name, keys, values)
 }
 
+# Create formal args that can be assigned to a function
+# based on the inspected Python signature.
+# @param pyParams the function info args as from getFunctionInfo
+.createFormalArgs <- function(pyParams) {
+  argNames <- pyParams$args
+  defaults <- pyParams$defaults
+
+  if (length(argNames) > 0 && argNames[1] == 'self') {
+    argNames <- argNames[-1]
+  }
+
+  newArgs <- setNames(rep(list(quote(expr =)), length(argNames)), argNames)
+
+  ## If there are no defaults
```

Comment:
Looks like this comment was copied from my original function but the code that follows was changed, so it doesn't match what's actually happening on the next line.